src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+public class Ui {
+    // Constants
+    public static final String SEPARATOR = "    ____________________________________________________________";
+    public static final String INDENT = "     "; // 5 spaces
+
+    static void welcome_greeting() {
```

Comment:
Perhaps could name method to be an action and use camelCase i.e. printWelcomeGreeting

Similar for exit_greeting method!